two_points_on_the_same_branch.sh

```diff
@@ -248,7 +253,7 @@ for coq_opam_package in $coq_opam_packages; do
             echo "Testing OLD commit"
         fi
 
-        eval $(opam config env)
+        eval $(opam env)
```

Comment:
What's wrong with opam config env? the manpage says `This is a shortcut, and equivalent to opam config env.`

Reply:
Both are OK, I guess I use it to remind myself I'm in an OPAM 2 setting.